tests/test_functional/fastapi_chat_completion_openai/server_openai.py

```diff
@@ -36,9 +35,16 @@ def create_agent(dto: CreateAgentDto):
         api_key=dto.api_key,
     )
 
-
 bridge = LangchainOpenaiApiBridgeFastAPI(app=app, agent_factory_provider=create_agent)
 bridge.bind_openai_chat_completion(prefix="/my-custom-path")
 
+def event_adapter(event):
+    kind = event["event"]
+    match kind:
+        case "on_chat_model_stream":
+            return event
+
+bridge.bind_openai_chat_completion(prefix="/my-custom-events-path", event_adapter=event_adapter)
+
```

Comment:
This seem to be dead code. Ideally the whole file should not have changes

Reply:
Oh, yep. That's dead code. I'll remove that.